doc/redirects.csv

```diff
@@ -39,3 +39,2737 @@ contributing_docstring,development/contributing_docstring
 developer,development/developer
 extending,development/extending
 internals,development/internals
+
+# api
+generated/pandas.api.extensions.ExtensionArray.argsort,../api/generated/pandas.api.extensions.ExtensionArray.argsort
```

Comment:
is the .. correct here?

Reply:
Yes, it's required. The redirect was initially absolute, but after the discussion, I made it relative, so to go from `generated/` to `api/generated/` the path is `../api/generated/`.

I tested that they work locally (got it wrong couple of times, that's why it took me a while to get it ready, generating the docs takes a while).

Reply:
OK, yes, because it is relative to the final page of course (I was confused because in the above it is the absolute path for both)